.travis/install-dev-vault.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -o errexit
+
+# Based on https://www.vaultproject.io/docs/platform/k8s/helm/run
```

Comment:
Great! please do the same with non dev vault server

Reply:
See #797